Specification.md

```diff
@@ -0,0 +1,17 @@
+### plugins.yaml
+
+The `plugins.yaml` file contains a list of objects, each representing a plug-in.
+
+YAML has been chosen because of its good readability while still being machine-readable and reasonably widespread. Compatibility with other formats is achieved through the CI, which converts the YAML file to other formats.
+
+### Plug-Ins
+
+A Plug-In object consists of the following key-value pairs:
+
+- `name`
+- `url`: A link to the project's github repository.
+- `version`: Preferably the name of a git tag, or a commit hash.
```

Comment:
Not currently. It's a nice feature, but i don't see it being worth the effort for now. I'd rather avoid second-system syndrome by doing this at some later point.

Reply:
However we go about it, either by compatibility tags on the plugin itself, or on the versions, or _____, we need some way for the generated plugin list to be filterable, so that the game client/esl2/both can avoid recommending plugins that use unsupported features or syntax.
This scenario won't be an issue for folks who are always using continuous builds (unless we drop some syntax support at some future time), but will affect players who install from steam/distro pkg/etc.